librz/util/str.c

```diff
@@ -3819,3 +3819,71 @@ RZ_API char *rz_str_version(const char *program) {
 }
 
 #undef RZ_STR_PKG_VERSION_STRING
+
+/**
+ * \brief Converts a raw buffer to a printable string based on the selected options
+ *
+ * \param  buf          The buffer to read
+ * \param  len          The length of the buffer
+ * \param  max_wrap_len The max line length when RZ_STR_STRINGIFY_WRAP is set
```

Comment:
i think this is not very clear. what does it do? (just to understand without me having to look at the implementation :D)